c/makeotf/makeotf_lib/source/hotconv/OS_2.c

```diff
@@ -345,7 +346,7 @@ int OS_2Fill(hotCtx g) {
     h->tbl.yStrikeoutSize = font->win.StrikeOutSize;
     h->tbl.yStrikeoutPosition = font->win.StrikeOutPosition;
 
-    h->tbl.sFamilyClass = CLASS_NONE; /* No classification */
+    // h->tbl.sFamilyClass = CLASS_NONE; /* No classification */
```

Comment:
Please remove this commented-out code.

Reply:
d'oh! Thanks for catching this. Fixed in https://github.com/adobe-type-tools/afdko/pull/1192/commits/a41e3f954209b80645cc16d1b418be167d92be91